install/elabftw.sql

```diff
@@ -346,7 +346,10 @@ INSERT INTO `config` (`conf_name`, `conf_value`) VALUES
 ('debug', '0'),
 ('lang', 'en_GB'),
 ('login_tries', '3'),
+('mail_from', ''),
+('mail_method', 'sendmail'),
```

Comment:
IDK, sendmail seems too restrictive. Maybe "Local MTA (default)" would be more correct.

Sendmail but also Postfix, Qmail or Exim could be installed. I wouldn't want someone with qmail thinking "oh no I can't use the sendmail method because I don't have sendmail installed".